<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentWindow</class>
 <widget class="QWidget" name="AddStudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Student</string>
  </property>
  <widget class="QListWidget" name="StudentList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>250</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="StudentAddList">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>250</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ClassSelector">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Choose a class</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="AddStudentsButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>355</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;---</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
